package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
```

Comment:
Please revert any changes to `package-lock.json`. It should not show up in the PR.

Reply:
Done